extensions/windows-event-log/ConsumeWindowsEventLog.h

```diff
@@ -79,6 +81,8 @@ class ConsumeWindowsEventLog : public core::Processor
   void unsubscribe();
   int processQueue(const std::shared_ptr<core::ProcessSession> &session);
 
+  void createTextOutput(std::wstringstream& strStream, MSXML2::IXMLDOMElementPtr pRoot, std::vector<std::wstring>& listTag);
```

Comment:
Could you make the pRoot param first and const, too?
The only thing I would like to achieve by this is the signature clearly differentiating in and out params.